src/routes/router.js

```diff
@@ -85,7 +88,7 @@ export default Router.extend({
   faq() { this.render(<FAQ/>) },
   subscribe(id) { this.render(<FirstSubscriptionPage planName={id}/>) },
   userShow(id) { this.render(<UserShow userId={id}/>, {backgroundColor: 'GREY'}) },
-  organizationShow(id) { this.render(<OrganizationShow organizationId={id} key={id}/>, {backgroundColor: 'GREY'}) },
+    organizationShow(id) { this.render(<OrganizationShow organizationId={id} key={id} tab={extractOrganizationTab(window.location.search)}/>, {backgroundColor: 'GREY'}) },
```

Comment:
Can that be removed as part of this story?  We'll have to do it some time, seems cleaner to ensure that each story ends up clean.